holo-bgp/src/packet/message.rs

```diff
@@ -130,6 +131,7 @@ pub enum Capability {
 #[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
 #[derive(EnumAsInner)]
 #[derive(Deserialize, Serialize)]
+#[derive(Arbitrary)]
```

Comment:
unnecessary derive

Reply:
`NegotiatedCapability` is a type inside a Treeset field in `DecodeCxt`:

```rust
    pub capabilities: BTreeSet<NegotiatedCapability>,
``` 
It therefore needs to be "arbitrarated" one way or another.

Reply:
`NegotiatedCapability` is needed in the `capabilities` field in the `DecodeCxt` struct. 

Since `DecodeCxt`  derives `Arbitary`, we need to derive the `Arbitrary` trait as well.